VRCFaceTracking/Params/Expressions/UnifiedExpressionsMerger.cs

```diff
@@ -0,0 +1,198 @@
+                "v2/EyesWiden"),
+            
+            #endregion
+
+            #region Eyes Squint (Combined)
+
+            new EParam(exp =>
+                exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].AdjustedWeight  > exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].AdjustedWeight
+                ? exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].AdjustedWeight
+                : exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].AdjustedWeight,
+                "v2/EyesSquint"),
+            
+            #endregion
+
+            #region Eye Expressions
+
+            new EParam(exp => exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].AdjustedWeight - exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].AdjustedWeight, "v2/LeftEyeExpression"),
+            new EParam(exp => exp.Shapes[(int)UnifiedExpressions.EyeWideRight].AdjustedWeight - exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].AdjustedWeight, "v2/RightEyeExpression"),
+
+            new EParam(exp =>
+                (exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].AdjustedWeight) / 2.0f -
+                (exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].AdjustedWeight) / 2.0f,
+                "v2/EyesExpression"),
+
+            
+            #endregion
+
+            #region Eyebrows Compacted
+
+            new EParam(exp => Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerUpRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterUpRight].AdjustedWeight) -
+                Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerDownRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterDownRight].AdjustedWeight),
+                "v2/RightBrowY"),
+
+            new EParam(exp => Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerUpLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterUpLeft].AdjustedWeight) -
+                Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerDownLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterDownLeft].AdjustedWeight),
+                "v2/LeftBrowY"),
+
+            new EParam(exp =>
+                (Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerUpRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterUpRight].AdjustedWeight) -
+                Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerDownRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterDownRight].AdjustedWeight)) +
+                (Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerUpLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterUpLeft].AdjustedWeight) -
+                Math.Min(1, exp.Shapes[(int)UnifiedExpressions.BrowInnerDownLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.BrowOuterDownLeft].AdjustedWeight)) / 2.0f,
+                "v2/LeftBrowY"),
```

Comment:
v2/BrowsY

Reply:
All right will make that change, thanks for catching!

Reply:
Still needs fixed